packages/backend/src/services/fanart.ts

```diff
@@ -0,0 +1,76 @@
+import { FanartDoc, FanartInterface, FanartModel } from '@model/fanart'
+
+/**
+ * Get all fanart
+ * @returns array of fanart document
+ */
+export const getAllFanarts = async (
+  lastId: string,
+  limit: number
+): Promise<FanartDoc[]> => {
+  // if lastId is not given or given an empty string
+  if (lastId === 'NULL' || lastId === '') {
+    return await FanartModel.find().limit(limit)
+  }
+  return await FanartModel.find({ _id: { $gt: lastId } }).limit(limit)
```

Comment:
I get a "redundant await" with these. maybe `exec()` is needed or did you intentionally call it like this?

Reply:
This also applies for the `message` counterpart

Reply:
yeah, This is a `redundant-await`
 I didn't notice because the `no-return-await` rule doesn't seem to exist in `eslintrc`